code/modules/food_and_drinks/food/foods/junkfood.dm

```diff
@@ -84,6 +84,14 @@
 	list_reagents = list("nutriment" = 2, "sugar" = 4)
 	tastes = list("bread" = 1)
 
+/obj/item/reagent_containers/food/snacks/stroopwafel
+	name = "stroopwafel"
+	desc = "Straight from the snack factories of Space Den Haag comes a classic dutch treat!"
+	icon_state = "stroopwafel"
+	filling_color = "#EDB878"
+	junkiness = 20
```

Comment:
Bit of a nitpick. If these are supposed to be premium and rare I dont think it really makes sense for them to have a high `junkiness`

Reply:
A good point!